libs/fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpapi/taxonomy/TaxonomyRsApiRestClientClient.kt

```diff
@@ -0,0 +1,87 @@
+package org.wordpress.android.fluxc.network.rest.wpapi.taxonomy
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import org.wordpress.android.fluxc.Dispatcher
+import org.wordpress.android.fluxc.generated.TaxonomyActionBuilder
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.model.TermModel
+import org.wordpress.android.fluxc.model.TermsModel
+import org.wordpress.android.fluxc.module.FLUXC_SCOPE
+import org.wordpress.android.fluxc.network.rest.wpapi.rs.WpApiClientProvider
+import org.wordpress.android.fluxc.store.TaxonomyStore.FetchTermsResponsePayload
+import org.wordpress.android.fluxc.store.TaxonomyStore.TaxonomyError
+import org.wordpress.android.fluxc.store.TaxonomyStore.TaxonomyErrorType
+import org.wordpress.android.fluxc.utils.AppLogWrapper
+import org.wordpress.android.util.AppLog
+import rs.wordpress.api.kotlin.WpRequestResult
+import uniffi.wp_api.TagListParams
+import uniffi.wp_api.TagWithEditContext
+import javax.inject.Inject
+import javax.inject.Named
+import javax.inject.Singleton
+
+@Singleton
+class TaxonomyRsApiRestClientClient @Inject constructor(
```

Comment:
This appears to include a duplicate `Client` name. Should this be deduplicated? 

Also, it appears we may diverge in how we case the Rust label. Some places us `RS`, others use `Rs`. We should likely align our practices.